en_ewt-ud-train.conllu

```diff
@@ -218561,7 +218561,7 @@
 13	forget	forget	VERB	VB	Mood=Imp|VerbForm=Fin	7	parataxis	7:parataxis	_
 14	to	to	PART	TO	_	15	mark	15:mark	_
 15	tell	tell	VERB	VB	VerbForm=Inf	13	xcomp	13:xcomp	_
-16	'em	they	PRON	PRP	Abbr=Yes	15	obj	15:obj	CorrectForm=them
+16	'em	they	PRON	PRP	Abbr=Yes|Case=Acc|Number=Plur|Person=3|PronType=Prs	15	obj	15:obj	CorrectForm=them
```

Comment:
Maybe this should be treated like *ya*, with `Style=Coll`.

Reply:
I've applied this change along with the check for the "'em" personal pronoun form.